components/Ink/src/private/MDCLegacyInkLayer.m

```diff
@@ -207,7 +209,8 @@ @implementation MDCLegacyInkLayerForegroundRipple
 
 - (void)setupRipple {
   CGFloat random = MDCLegacyInkLayerRandom();
-  self.radius = (CGFloat)(0.9f + random * 0.1f) * kInkLayerForegroundRadiusGrowthMultiplier;
+  self.radius =
+      (CGFloat)((CGFloat)0.9 + random * (CGFloat)0.1) * kInkLayerForegroundRadiusGrowthMultiplier;
```

Comment:
is the outside CGFloat casting still needed after the change?

Reply:
The inner ones are probably not necessary, but I'd generally prefer to avoid making changes I don't have to. This was supposed to be an automated PR.